packages/gatsby-plugin-gatsby-cloud/src/components/buttons/InfoIndicatorButton.js

```diff
@@ -39,14 +88,12 @@ export default function InfoIndicatorButton(props) {
       name: `info hover`,
     })
   }
-
   return (
     <IndicatorButton
       testId="info"
-      iconSvg={infoIcon}
-      onMouseEnter={buttonProps?.active && trackHover}
+      iconSvg={buttonProps?.showInfo ? infoIconActive : infoIcon}
+      onClick={buttonProps?.active && trackHover}
```

Comment:
Does this make sense? We're tracking the hover when the button is being clicked now?